hbase-server/src/main/java/org/apache/hadoop/hbase/mob/MobConstants.java

```diff
@@ -91,12 +80,46 @@
   public static final int DEFAULT_MOB_COMPACTION_CHORE_PERIOD =
     24 * 60 * 60 * 7; // a week
   public static final String MOB_COMPACTOR_CLASS_KEY = "hbase.mob.compactor.class";
+
+  /**
+   * Mob compaction type: "full", "io_optimized"
+   * "full" - run full major compaction (during migration)
+   * "io_optimized" - optimized version for use  case with infrequent updates/deletes
```

Comment:
the actual value below is "optimized" either update the javadoc or update the value.